### PR TITLE
Add Dashboard Access and Profiling Page

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Add TiDB Dashboard Overview and Instance Profiling Page.

I'll add all TiDB Dashboard related documents in TOC when most of them are merged into docs.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3244 https://github.com/pingcap/docs-cn/pull/3440
- Other reference link(s):

### CODE REVIEW

**Comment by TomShawn** [comment]
> [pingcap/docs-cn#3440](https://github.com/pingcap/docs-cn/pull/3440) made some little changes to the Access page. Do we need to wait for that PR being merged first?

OK

**Comment by TomShawn** [comment]
@breeswish I'll create another PR to align the changes made in https://github.com/pingcap/docs-cn/pull/3440. Can we merge this PR first? coz several Dashboard en PRs cannot be merged until this one is merged. All changes of images are in this PR. PTAL again, thanks!

File: dashboard/dashboard-access.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+- Chrome >= 77
+- Firefox >= 68
+- Edge >= 17
+
+> **Note:**
+>
+> If you use the browsers above of earlier versions or other browsers to access TiDB Dashboard, some interfaces might not work properly.
+
+## Login
```
Comment (breeswish) [inline dashboard/dashboard-access.md:27]:
```suggestion
## Sign in
```

Maybe better to align with the Sign In title in UI?

File: dashboard/dashboard-access.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+For the first-time access, TiDB Dashboard displays the user login interface, as shown in the image below. You can log in using the TiDB `root` account.
```
Comment (breeswish) [inline dashboard/dashboard-access.md:29]:
```suggestion
For the first-time access, TiDB Dashboard displays the user sign in interface, as shown in the image below. You can sign in using the TiDB `root` account.
```

File: dashboard/dashboard-access.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+You can use TiDB Dashboard in the following desktop browsers which are common and updated timely:
+
+- Chrome >= 77
+- Firefox >= 68
+- Edge >= 17
+
+> **Note:**
+>
+> If you use the browsers above of earlier versions or other browsers to access TiDB Dashboard, some interfaces might not work properly.
```
Comment (breeswish) [inline dashboard/dashboard-access.md:25]:
```suggestion
> If you use the browsers above of earlier versions or other browsers to access TiDB Dashboard, some functionalities might not work properly.
```

File: dashboard/dashboard-access.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+![Login interface](/media/dashboard/dashboard-access-login.png)
+
+If the one of the following situations exists, the login might fail:
+
+- TiDB `root` user does not exist.
+- PD is not started or cannot be accessed.
+- TiDB is not started or cannot be accessed.
+- Wrong `root` password.
+
+Once you have logged in, the login status remains within the next 24 hours. To learn how to log out, refer to the [Log out](#log-out) section.
```
Comment (breeswish) [inline dashboard/dashboard-access.md:40]:
```suggestion
Once you have signed in, the session remains valid within the next 24 hours. To learn how to sign out, refer to the [Logout](#logout) section.
```

File: dashboard/dashboard-access.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+## Switch language
+
+The following languages are supported in TiDB Dashboard:
+
+- Chinese (simplified)
+- English
+
+In the login interface, you can click the **Switch Language** drop-down box to switch the interface language.
```
Comment (breeswish) [inline dashboard/dashboard-access.md:49]:
```suggestion
In the sign in page, you can click the **Switch Language** drop-down box to switch the interface language.
```

File: dashboard/dashboard-profiling.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+category: how-to
+---
+
+# Instance Profiling Page
+
+The instance profiling page summarizes the internal performance data of each TiDB instance, TiKV instance, and PD instance. With this page, you can view the instance profiling without restarting these instances.
+
+The summarized performance data can be displayed as a flame graph or a directed acyclic graph, which visually shows the internal operations and proportions performed on the instances during the performance summary period. With this graph, you can quickly learn the main CPU resource consumption of these instances.
+
+## Access instance profiling page
```
Comment (breeswish) [inline dashboard/dashboard-profiling.md:13]:
```suggestion
## Access the page
```

File: dashboard/dashboard-profiling.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+You can access the instance profiling page using one of the following methods:
+
+- After logging into TiDB Dashboard, click **Advanced Debugging** → **Profile Instances** on the left navigation bar.
+
+  ![Access instance profiling page](/media/dashboard/dashboard-profiling-access.png)
+
+- visit <http://127.0.0.1:2379/dashboard/#/instance_profiling> in your browser. Replace `127.0.0.1:2379` with the actual PD instance address and port.
```
Comment (breeswish) [inline dashboard/dashboard-profiling.md:21]:
```suggestion
- Visit <http://127.0.0.1:2379/dashboard/#/instance_profiling> in your browser. Replace `127.0.0.1:2379` with the actual PD instance address and port.
```

File: dashboard/dashboard-access.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+category: how-to
+---
+
+# Access TiDB Dashboard
+
+To access TiDB Dashboard, visit <http://127.0.0.1:2379/dashboard> via your browser. Replace `127.0.0.1:2379` with the actual PD instance address and port.
+
+> **Note:**
+>
+> If you have deployed multiple PD components in your cluster, you can use the address of only one PD component to access TiDB Dashboard, because the Dashboard operates on only one PD component. If you access via other PD components, your browser is redirected to that PD component. Therefore, if the firewall is not configured for this instance on which PD operates, you might not be able to access TiDB Dashboard.
```
Comment (lilin90) [inline dashboard/dashboard-access.md:13]: Please check [the latest Chinese version](https://docs.pingcap.com/zh/tidb/dev/dashboard-access#%E5%A4%9A-pd-%E5%AE%9E%E4%BE%8B%E8%AE%BF%E9%97%AE) and update it.

File: dashboard/dashboard-access.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+
+## Browser compatibility
+
+You can use TiDB Dashboard in the following desktop browsers which are common and updated timely:
```
Comment (lilin90) [inline dashboard/dashboard-access.md:17]:
```suggestion
You can use TiDB Dashboard in the following common desktop browsers of a relatively newer version:
```

File: dashboard/dashboard-access.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+>
+> If you use the browsers above of earlier versions or other browsers to access TiDB Dashboard, some functions might not work properly.
+
+## Sign in
+
+For the first-time access, TiDB Dashboard displays the user sign in interface, as shown in the image below. You can sign in using the TiDB `root` account.
+
+![Login interface](/media/dashboard/dashboard-access-login.png)
+
+If the one of the following situations exists, the login might fail:
```
Comment (lilin90) [inline dashboard/dashboard-access.md:33]:
```suggestion
If one of the following situations exists, the login might fail:
```

File: dashboard/dashboard-access.md [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+- Wrong `root` password.
+
+Once you have signed in, the session remains valid within the next 24 hours. To learn how to sign out, refer to the [Logout](#logout) section.
+
+## Switch language
+
+The following languages are supported in TiDB Dashboard:
+
+- Chinese (simplified)
+- English
```
Comment (lilin90) [inline dashboard/dashboard-access.md:47]:
```suggestion
- English
- Chinese (simplified)
```

File: dashboard/dashboard-access.md [hunk 12]
```diff
@@ -0,0 +1,57 @@
...
+
+In the **sign in** page, you can click the **Switch Language** drop-down box to switch the interface language.
```
Comment (lilin90) [inline dashboard/dashboard-access.md:49]:
```suggestion
In the **SQL User Sign In** page, you can click the **Switch Language** drop-down list to switch the interface language.
```

File: dashboard/dashboard-access.md (hunk 8 above)
Comment (TomShawn) [reply dashboard/dashboard-access.md:13]: The updated part contains links of several documents added in https://github.com/pingcap/docs-cn/pull/3440. For convenience, I'll update the change made to `dashboard-access.md` in another PR together with those newly added documents.

Comment (lilin90) [reply dashboard/dashboard-access.md:13]: Got it.

**Pull request merged**